openlibrary/i18n/fr/messages.po

```diff
@@ -13,7 +13,7 @@ msgstr ""
 "Last-Translator: Julien Lepiller <julien@lepiller.eu>\n"
 "Language: fr\n"
 "Language-Team: fr <traduc@traduc.org>\n"
-"Plural-Forms: nplurals=2; plural=(n > 1)\n"
+"Plural-Forms: nplurals=2; plural=(n >= 2)\n"
```

Comment:
Now I see… English pluralizes 1 1/2 but French does not.
```python
I have an apple
You have two apples
Joe has one and a half apples
——
J'ai une pomme
tu as deux pommes
Joe a une pomme et demie
```

Reply:
Unicode pluralization rules confirm this although their notation is a little ambiguous about whether the breakpoint is at 1,5 or 2,0
https://www.unicode.org/cldr/cldr-aux/charts/29/supplemental/language_plural_rules.html#fr

Reply:
Well shoot. It figures there would be a more authoritative source, @tfmorris. I didn't even think to check there. I also see how it's not super clear where the breakpoint is.

I tried looking in French grammar books here on Open Library, but I confess my search wasn't as thorough as it might have been. I will set aside some time later to try to find something dispositive of the matter.

Reply:
It's a nit, so I wouldn't worry about it too much. I suspect the actual breakpoint is 2,0 and that it's just notational weirdness in Unicode site.

Reply:
As much as I make mistakes all day long, that doesn't stop me from being vaguely obsessive about getting things 'right'. However, it also doesn't stop me from throwing in the towel. I have thrown in the towel. :D

Having failed to find something I would say is definitive, I am going to hope a bunch of random people making posts on various forums (fora?) claiming to know about French grammar on the interweb haven't let me down. And that confirmation bias doesn't have me *too* tightly in its grasp.